### ISSUE TITLE
OSX: Signing out and back end clears chat history

### ISSUE THREAD

**Issue by dustinfarris** [keybase/client#9425]
If I sign out, then sign back in on Mac, all the chat history is gone.  I can still see it on my iPhone though.

my log id: 206a9b1522e292e16110fd1c

**Comment by NicoleRauch** [comment]
Yep, same for me, it's gone in the Mac app but still visible on the phone.
